### DISCUSSION TITLE
fidget插件 字体颜色如何配置

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [nshen/learn-neovim-lua#88] [Q&A]
![图片](https://user-images.githubusercontent.com/43649186/184306745-6ca442dc-3b0e-4a1a-9651-da2f6fb46be5.png)
这个字体颜色太黑了，想变成白色

**Comment by nshen** [comment]
跟我的 tokyonight 主题刚刚好
<img width="516" alt="image" src="https://user-images.githubusercontent.com/181506/184468568-7f538b28-7738-4d30-8dd3-63a97b5e8d01.png">

颜色我也没研究过，看帮助大概是调整这两个值，你可以试试

```
highlight FidgetTitle ctermfg=110 guifg=#6cb6eb
```
